package.json

```diff
@@ -20,11 +20,12 @@
     "eslint-config-next": "13.4.7",
     "next": "13.4.7",
     "postcss": "8.4.24",
+    "prop-types": "^15.8.1",
```

Comment:
I don't think we need this library; we use TypeScript for such things.

Reply:
removed